### ISSUE TITLE
Run tests under .NET Core 2.1

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1361]
We currently run tests under .NET Framework 4.6.1 and .NET Core 1.0.
Sometimes newer platforms introduce changes in behaviour, as noted in #1345. Even though that one bug was resolved, we should test under the newer .NET Core to ensure that everything else works (and continues to work).

**Comment by silkfire** [comment]
Version 4.8.0 doesn't seem to be compatible with .NET Core 2.1, but 4.7.1 is. What figures?

**Comment by blairconrad** [comment]
Hi, @silkfire. I'm unaware of the deficit. In what way do you find 4.8.0 not to be compatible with .NET Core 2.1?

**Comment by silkfire** [comment]
@blairconrad NuGet Package Manager says that it failed to install version 4.8.0 due to incompatible framework (netcore2.1) when trying to upgrade.

**Comment by thomaslevesque** [comment]
Hi @silkfire,

Seems to work fine for me:

```
> dotnet new console -f netcoreapp2.1
> dotnet add package FakeItEasy dotnet add package FakeItEasy -v 4.8.0
```

No error, the package is added and the project builds fine.

Looks like a NuGet bug...

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 4.9.0](https://github.com/FakeItEasy/FakeItEasy/releases/4.9.0).
